v1.1/admin_ui_cluster_overview.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cluster Overview 
+toc: false
+feedback: false
+---
+
+CockroachDB’s Admin UI enables you to monitor details metrics such as SQL Queries, Service Latency, Replicas per Node, and Capacity of your cluster.
+
+<div id="toc"></div>
+
+On [accessing the Admin UI](explore-the-admin-ui.html#access-the-admin-ui), the first dashboard you see by default is the Cluster Overview. 
+
+#### Cluster and Node View for the Time-Series Graphs
+By default, the time series panel displays the cluster view, which shows the metrics for the entire cluster. For every time series data, the visualization for the entire cluster is the sum of the metric for each node. 
```

Comment:
"every time series data" I think shouldn't have the word "data".  Also, the cluster-level visualization doesn't always show the sum - some charts show the metric broken out by node.  Each individual metric makes that clear, but this summary doesn't.